onnx/defs/logical/defs.cc

```diff
@@ -121,6 +121,34 @@ ONNX_OPERATOR_SET_SCHEMA(
             {"tensor(bool)"},
             "Constrains output to boolean tensor."));
 
+ONNX_OPERATOR_SET_SCHEMA(
+	LessOrEqual,
+	9,
+	OpSchema()
+		.FillUsing(BinaryLogicDocGenerator("less or equal"))
+		.TypeConstraint(
+			"T",
+			OpSchema::all_numeric_types(),
+			"Constarins input to float tensors.")
```

Comment:
The comment does not match the types allowed. Also nit: spelling for "constrains" .

Reply:
Yeah, and I just noticed the schema of Greater and Less were also wrong here.